signale.js

```diff
@@ -155,6 +155,10 @@ class Signale {
     return meta;
   }
 
+  _prefixSufixNestedTernary(prefix, suffix, args) {
```

Comment:
Awesome! A couple of minor tweaks and it is ready to be merged! 🎉 

```js
  _hasAdditional({suffix, prefix}, args) { // Rename to something simpler, for example: _hasAdditional() 
    // Use destructuring directly on function definition
    return (suffix || prefix) ? '' : util.format(...args); // Add simple parenthesis around condition to make it clearer
  }
```

Reply:
It's better than previous